### DISCUSSION TITLE
An error with MRTK and Unity

### DISCUSSION THREAD

**Discussion by galegozi** [microsoft/MixedRealityToolkit-Unity#9565] [Q&A] [answered]
Hi,
I was using Unity 2019.4.18 and 2019.4.12 and had problems using MRTK with both Unity versions.
Here is a screenshot showing one of the errors I had in Unity 2019.4.18.
![image](https://user-images.githubusercontent.com/8800898/112709722-b992c700-8e89-11eb-9d55-07f31ed69c4e.png)
Thanks,
Gal

**Comment by david-c-kline** [comment, marked as answer]
Hi @galegozi,

Sorry you are hitting an error. This sounds like the files were corrupted when they were downloaded (likely not a complete file). Can you redownload the updated (1.0.2103.3 Beta) Feature Tool from https://aka.ms/mrfeaturetool and try again?

I would recommend deleting the contents of c:\users\<yourname>\AppData\Loca\MixedRealityFeatureTool\PackageCache before trying to download the MRTK.

If you still have issues, this version generates a diagnostic log that gets saved to  c:\users\<yourname>\AppData\Loca\MixedRealityFeatureTool that we can use to help diagnose the issue.

Thanks!

**Reply by galegozi** [reply]
Hi @davidkline-ms,
I was able to solve the issue, thank you for your help!

Also, can you please add me to the Microsoft GitHub (I see that it says that you are a member of Microsoft)? I am starting on April 19?
Thanks,
Gal
